backend/src/lib.rs

```diff
@@ -59,7 +60,7 @@ pub fn router(pool: SqlitePool) -> Result<Router, Box<dyn Error>> {
 
     let app = Router::new()
         .nest("/api/elections", election_routes)
-        .nest("/api/polling_stations", polling_station_routes);
+        .nest("/api/polling_stations", data_entry_routes);
```

Comment:
This now looks a bit inconsistent. Maybe we can change the url to include the `/:polling_station_id/data_entries/` here, or keep the polling_station_routes and nest data_entry_routes in there again?

Reply:
Good idea, changed in 19ae4988967a1f8cb48aa3453b42e6679e129e6e.